lib/rmg/modificators/ObjectManager.cpp

```diff
@@ -86,6 +86,18 @@ void ObjectManager::updateDistances(const rmg::Object & obj)
 	}
 }
 
+void ObjectManager::updateDistances(const int3 & pos)
+{
+	RecursiveLock lock(externalAccessMutex);
+	tilesByDistance.clear();
+	for (auto tile : zone.areaPossible().getTiles()) //don't need to mark distance for not possible tiles
+	{
+		ui32 d = pos.dist2dSQ(tile); //optimization, only relative distance is interesting
+		map.setNearestObjectDistance(tile, std::min(static_cast<float>(d), map.getNearestObjectDistance(tile)));
+		tilesByDistance.push(std::make_pair(tile, map.getNearestObjectDistance(tile)));
+	}
+}
```

Comment:
This code is pure copy-paste of method above, with changes in a single line. Is it possible to avoid somehow?

Reply:
I could wrap the line in lambda, but it would be very artifical, inlexible and wouldn't make the total lines of code any smaller.